src/Stepper/index.js

```diff
@@ -1,5 +1,5 @@
+export Step from './Step';
+export StepButton from './StepButton';
+export StepContent from './StepContent';
+export StepLabel from './StepLabel';
 export Stepper from './Stepper';
```

Comment:
Should this be `export default from './Stepper'` and the imports in he examples be changed to use the default import? I think that's the pattern we're using elsewhere.